### DISCUSSION TITLE
New Release, New Interface & New Release naming

### DISCUSSION THREAD

**Discussion by immauss** [immauss/openvas#305] [Announcements]
OK ... 
The release numbering change is LONG overdue. From now on, tags will follow the date. GB had at one time long ago referred to their GVM releases by the gvmd release version and only updated once or twice a year (sometimes every other year.)  Well, they have been through 23.x.x for some things and are now on 24.x.x for some things .... So ... time for change here too. 

From now on, release tags will follow the  %y.%m.{serial #}  format.  ( ie YY.MM.<sequential number> )

If there is a Major change, I'll announce here. 

In this release, there is one noticeably major change. GSA received an interface update. (All from GB .. I only updated the login page.... ) 

I want to thank ALL of my paid subscribers and the folks who open issues and help me resolve bugs. You folks are ALL awesome and VERY appreciated. 

If you are using this container as part of your current bussiness model or even personally on a regular basis, please consider a subscription as it will give you access to the most up-to-date feed data, an actual SLA for problems and increased influence on the future direction of the project. 

Thank you,
Scott

<hr /><em>This discussion was created from the release <a href='https://github.com/immauss/openvas/releases/tag/24.10.1'>New Release, New Interface & New Release naming</a>.</em>

**Comment by mshokarimo7** [comment]
Thank you so much for making a docker image and maintaining it up to this day!. Seems to me that once GB had thought to have resolved the issue, they forgot about the thread. Or the mods that were looking at the threads got a new job elsewhere lol. Anywho, its people like you who keep the internet sane and working))

**Comment by Paiii3** [comment]
To remind, your certificate was expired.
![image](https://github.com/user-attachments/assets/784257ba-536e-4fc0-a3ef-0f8a01a55a12)

**Comment by immauss** [comment]
Thanks. Something went wonky with certbot. It's all fixed now. 

Thanks,
Scott
